lib/cadet/assessments/assessments.ex

```diff
@@ -703,8 +703,9 @@ defmodule Cadet.Assessments do
       {:error, :race_condition} ->
         {:error, {:internal_server_error, "Please try again later."}}
 
-      {:error, :vote_not_unique} ->
-        {:error, {:bad_request, "Invalid vote or vote is not unique! Vote is not saved."}}
+      # Description changed from :vote_not_unique and corresponding message changed.
```

Comment:
It is Okay to remove the comment, since other developer might be buffled by the word "change", since they might not be following this project. This change history is captured in github so it is okay.